### ISSUE TITLE
Lighthouse report: Accessibility concerns

### ISSUE THREAD

**Issue by msfjarvis** [lxndrblz/anatole#17]
Running Lighthouse on the site points out a few problems, which I'm splitting across separate issues based on category to make them easier to triage.

For accessibility, the following issues were reported:

- List items are not contained within `<ul>` or `<ol>` parent elements.

This seems to be violated by this line https://github.com/lxndrblz/anatole/blob/292a8afde384c8ce887f41cc80b94d77b3514879/layouts/partials/navbar.html#L5

- Background and foreground colors do not have sufficient contrast ratio

This is a concern I personally felt as well. The text color in dark mode needs to be a bit brighter to improve contrast and legibility. Once it is closer to white than gray we should be good to go.

[View the report](https://bin.msfjarvis.dev/~5ec7bcd61de3ad00122e4ef6/raw?ctype=text/html)

**Comment by msfjarvis** [comment]
Looking closely at a fresh Lighthouse report after enabling social icons, I noticed an additional set of errors that are fixed by #19

**Comment by lxndrblz** [comment]
Thanks Harsh, I just also ran the lighthouse test on my personal site. While all other categories were 100, accessibility lacked far behind. I'll review your pull request and go ahead and merge it. Thanks for your continuous efforts!

**Comment by msfjarvis** [comment]
Accessibility on my personal site is now at 100!
